libvips/include/vips/foreign.h

```diff
@@ -676,6 +676,19 @@ int vips_niftiload_source( VipsSource *source, VipsImage **out, ... )
 int vips_niftisave( VipsImage *in, const char *filename, ... )
 	__attribute__((sentinel));
 
+int vips_jp2kload( const char *filename, VipsImage **out, ... )
+	__attribute__((sentinel));
+int vips_jp2kload_buffer( void *buf, size_t len, VipsImage **out, ... )
+	__attribute__((sentinel));
+int vips_jp2kload_source( VipsSource *source, VipsImage **out, ... )
+	__attribute__((sentinel));
+int vips_jp2ksave( VipsImage *in, const char *filename, ... )
+	__attribute__((sentinel));
+int vips_heifsave_buffer( VipsImage *in, void **buf, size_t *len, ... )
```

Comment:
```suggestion
int vips_jp2ksave_buffer( VipsImage *in, void **buf, size_t *len, ... )
```
(should fix the failing CI build, I think)

Reply:
why is copy-paste so hard :(